oauthdoctor/diag/config_test.go

```diff
@@ -242,6 +244,53 @@ func TestPrint(t *testing.T) {
 	}
 }
 
+func TestReplaceConfig(t *testing.T) {
+	log.SetOutput(ioutil.Discard)
+
+	now := time.Now().Format("2006-01-02_")
+	dir, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("Error getting current dir: %s", err)
+	}
+
+	test := struct {
+		desc   string
+		cfg    ConfigFile
+		backup string
+	}{
+		desc: "Config file and backup file exist",
+		cfg: ConfigFile{
+			Filepath: filepath.Join(dir, "testdata"),
+			Filename: "python_config2",
+			Lang:     "python",
+		},
+		backup: "diag/testdata/python_config2_" + now,
+	}
+
+	backup := test.cfg.ReplaceConfig(DevToken, "randomToken")
+
+	config := test.cfg.GetFilepath()
+	_, err = os.Stat(config)
+	if os.IsNotExist(err) {
```

Comment:
Do you need a compound statement? Would the error explain the reason for the error?

Reply:
You are right. The compound statement is unnecessary. Here's the output from my quick test by printing err when a file doesn't exist:

`stat /tmp/testssssss: no such file or directory`